### PR TITLE
Add more assessment tools

### LINKED ISSUE #8: Add more tools to Kali image
# 🚀 Feature Proposal #

The assessment folks have requested more tools be added to the Kali image.  The requested tools are:
- [x] aquatone
- [x] CACTUSTORCH
- [x] checkpwnedemails
- [x] demiguise
- [x] dns-profile-randomizer-master
- [x] docker
- [x] docker-compose
- [x] Empire
- [x] ghostpack
- [x] gnmap-parser
- [x] Gobuster
- [x] Internal-Monologue
- [x] kerberoast
- [x] mikto
- [x] mimikatz
- [x] misc
- [x] mitm6
- [x] morphHTA
- [x] MS17-010
- [x] Neo4J
- [x] nlzr
- [x] nikto
- [x] pip
- [x] PowerSploit
- [x] PowerTools
- [x] PowerUpSql
- [x] RandomPS-Scripts
- [x] Responder
- [x] SecLists
- [x] SessionGopher
- [x] SharpShooter
- [x] Sublist3r
- [x] TikiTorch
- [x] Veil

## Motivation ##

The assessment folks need these tools to perform their assessments.

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description

This pull request adds more assessment tools to the Kali image, and resolves #8.

## 💭 Motivation and Context

The assessors require these tools to perform their job function.

## 🧪 Testing

I have used these changes to build and deploy a new Kali AMI to our COOL staging environment.  _(Not yet, but soon.)_

## 🚥 Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: molecule/default/requirements.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+- src: https://github.com/cisagov/ansible-role-pip
+  name: pip
+  version: bugfix/add-kali-specific-vars
```
Comment (dav3r) [inline molecule/default/requirements.yml]: Do we want to log an issue to remove `version: bugfix/add-kali-specific-vars` after https://github.com/cisagov/ansible-role-pip/pull/12 is merged?

File: tasks/aquatone.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    - name: Create a directory where aquatone will be installed manually
+      file:
+        group: "{{ group }}"
+        # Allow both root and the desired group to have full
+        # permissions in these directories
+        mode: 0775
+        path: /tools/{{ item.dir_name }}
+        state: directory
+      loop:
+        - {dir_name: "aquatone", repo: "https://github.com/michenriksen/aquatone/releases/download/v1.7.0/aquatone_linux_amd64_1.7.0.zip"}
```
Comment (dav3r) [inline tasks/aquatone.yml]: Can we make this repo URL (or maybe just the version #) into a variable?

**Review by dav3r — APPROVED** [review]
This is excellent!!  I had a couple of questions, but nothing too earth-shaking.
🎻

File: molecule/default/requirements.yml (hunk 1 above)
Comment (jsf9k) [reply molecule/default/requirements.yml]: No, I don't need an issue for that.

Comment (jsf9k) [reply molecule/default/requirements.yml]: Famous last words...

File: tasks/aquatone.yml (hunk 2 above)
Comment (jsf9k) [reply tasks/aquatone.yml]: Yes.  See commit 19cd876.

Comment (dav3r) [reply tasks/aquatone.yml]: 👍

**Review by mcdonnnj — APPROVED** [review]
I'm digging it, strong work 💪

**Review by felddy — APPROVED** [review]
So many tools in that toolbox:  🧰 
Strong work! 🛠️

File: tasks/aquatone.yml [hunk 3]
```diff
@@ -18,8 +18,7 @@
         mode: 0775
         path: /tools/{{ item.dir_name }}
         state: directory
-      loop:
-        - "{{ aquatone }}"
+      loop: "{{ aquatone }}"
```
Comment (jsf9k) [inline tasks/aquatone.yml:21]: Thanks for cleaning up the trash I left behind @mcdonnnj!

**Pull request merged**